### ISSUE TITLE
[Coursera] Neural Networks for Machine Learning

### ISSUE THREAD

**Issue by hyunjong-lee** [hyunjong-lee/studies#2]
Difficult Parts

**Comment by hyunjong-lee** [comment]
- Week 8: Q1, Q2
- Week 9: Softmax backpropagation derivation
- Week 12: Did not understand whole lecture
